includes/pixel.php

```diff
@@ -105,7 +105,6 @@ function wprtt_get_referring_page_title( $url ) {
 	}
 }
 
-// grab our site icon and redirect to it once the script finishes
-$site_icon_url = get_site_icon_url( 150 );
-wp_safe_redirect( $site_icon_url, 303 );
+header( 'Content-Type: image/png' );
+echo base64_decode( 'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABAQMAAAAl21bKAAAAA1BMVEUAAACnej3aAAAAAXRSTlMAQObYZgAAAApJREFUCNdjYAAAAAIAAeIhvDMAAAAASUVORK5CYII=' );
```

Comment:
This should have a comment explaining that it's a 1px transparent .gif

Reply:
That's right, added in 719fcba3a38f3e6bfc765d30e0c83f0dc6f0d887